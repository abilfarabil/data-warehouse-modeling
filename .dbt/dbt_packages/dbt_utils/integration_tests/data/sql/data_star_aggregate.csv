version https://git-lfs.github.com/spec/v1
oid sha256:78fa84a4ef02f95d6ee612bddaa3ac056b4927052a748109a39aa724f145cc47
size 67
