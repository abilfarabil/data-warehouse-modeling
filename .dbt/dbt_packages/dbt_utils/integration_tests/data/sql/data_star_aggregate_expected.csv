version https://git-lfs.github.com/spec/v1
oid sha256:296ab19387cf688cc940d4b90463d3d3e5da22fbe498a18cc4646642e82c22ac
size 64
